laravel/v3/auth/database/configuration.md

```diff
@@ -486,6 +486,24 @@ You may also add multiple if you'd prefer, or combine them with `key => value` p
 ],
 ```
 
+You may also define a closure directly:
+
+```php
+use App\Models\User as DatabaseUser;
+use App\Ldap\User as LdapUser;
+
+// ...
+'database' => [
+    // ...
+    'sync_attributes' => [
+        'name' => 'cn',
+        function (LdapUser $ldap, DatabaseUser $database) {
+            $database->email = $ldap->getFirstAttribute('mail');
+        },
```

Comment:
I'm not sure if we should advertise this in the docs, as this will cause your config to be unserializable and will throw an exception when executing `php artisan config:cache`:

<img width="836" alt="Screenshot 2023-06-23 at 9 46 23 AM" src="https://github.com/DirectoryTree/LdapRecord-Docs/assets/6421846/61c9aece-c1bb-41a0-a976-d9160300d217">

Reply:
Oh... that's a good point. The work it'd take to make it cache-able wouldn't be worth it. Seeing how it's accomplished when queueing a closure, it takes more than this is worth.